### PR TITLE
Trim whitespace from argument names in Function constructor

### PR DESCRIPTION
**Pull request by dbatyai**
JerryScript-DCO-1.0-Signed-off-by: Dániel Bátyai dbatyai.u-szeged@partner.samsung.com

### CODE REVIEW

File: jerry-core/ecma/base/ecma-helpers-string.cpp [hunk 1]
```diff
@@ -1880,6 +1881,82 @@ ecma_string_substr (const ecma_string_t *string_p, /**< pointer to an ecma strin
...
+
+  lit_utf8_size_t buffer_size = ecma_string_get_size (string_p);
+
+  if (buffer_size > 0)
+  {
+    MEM_DEFINE_LOCAL_ARRAY (utf8_str_p, buffer_size, lit_utf8_byte_t);
+    ecma_string_to_utf8_string (string_p, utf8_str_p, (ssize_t) buffer_size);
+
+    lit_utf8_iterator_t front = lit_utf8_iterator_create (utf8_str_p, buffer_size);
+    lit_utf8_iterator_seek_bos (&front);
```
Comment (sand1k) [inline jerry-core/ecma/base/ecma-helpers-string.cpp]: I think, line 1901 could be removed, as iterator is at the begin of string after creation.

Comment (dbatyai) [inline jerry-core/ecma/base/ecma-helpers-string.cpp]: Done.

**Comment by sand1k** [comment]
LGTM.

**Comment by kkristof** [comment]
Looks good to me.

**Pull request merged**